alpine-logstash-efset/logstash.conf

```diff
@@ -8,13 +8,43 @@ input {
 }
 
 filter {
+
+  if [channel] == "log" {
+
+    json {
+      source => "[data][performanceInfo]"
+      target => "performance"
+      remove_field => "[data][performanceInfo]"
+    }
+
+    if [data][resourceTiming] =~ /^\[/ {
+
+      json {
+        source => "[data][resourceTiming]"
+        target => "resource"
+        remove_field => "[data][resourceTiming]"
+        remove_field => "headers"
+      }
+
+      split {
+        field => "resource"
+      }
+    }
+  }
+
+  geoip {
+    source => "ip_address"
+    remove_field => "ip_address"
+  }
+
 }
 
 output {
-  if [type] == "http" {
+
+  if [channel] == "log" {
```

Comment:
I know this is not ideal, currently we have :

filebeat 
- type : log
- channel : -

log-backend
- type : logs
- channel : log

I didn't want to use 'type' because log & logs are to generic